README.md

```diff
@@ -152,6 +153,54 @@ await pool.end();
 connector.close();
 ```
 
+### Using with SQL Server
+
+Here is how to start a new
+[`tedious`](https://www.npmjs.com/package/tedious) connection.
+
+```js
+const {Connection, Request} = require('tedious');
+const {Connector} = require('@google-cloud/cloud-sql-connector');
+
+const connector = new Connector();
+const clientOpts = await connector.getTediousOptions({
+  instanceConnectionName: process.env.SQLSERVER_CONNECTION_NAME,
+  ipType: 'PUBLIC'
+});
+const connection = new Connection({
+  server: '0.0.0.0',
```

Comment:
I agree with Eno, we need to point out to users via a comment at the very least to leave these values as is. Otherwise users might try and change these values.

Reply:
oh that's a great point! alright let me see if I can come up with some nice wording for that